main.go

```diff
@@ -410,6 +423,37 @@ func run(c *cli.Context) error {
 		return fmt.Errorf("Error: %s\n", err)
 	}
 
+	rolloutCheck := c.StringSlice("rollout_check")
+	rolloutTimeout := c.Int("rollout_timeout")
+
+	log(fmt.Sprintf("Running rollout check for %v\n", rolloutCheck))
+
+	for _, deployment := range rolloutCheck {
+		command := []string{
+			"-t",
+			strconv.Itoa(rolloutTimeout),
+			kubectlCmd,
+			"rollout",
+			"status",
+			"deploy",
+			deployment,
```

Comment:
I think the output from running these in series could be confusing -- I would wonder why the other deployments aren't showing up. maybe adding & to the end of the command, or otherwise running in parallel, would be more intuitive.

Reply:
We should tackle this in a follow up.

IMO checking that the deployment succeeded should occur in another step.
This allows the user to configure run as many steps as there are deployments, and [group these steps](http://docs.drone.io/pipelines/#parallel-execution) so they run in parallel.

Of course the downside of this is the number of resources being rolled out in each kind of event would need to be known ahead of time (unless that step was a configurable plugin that could parse `.kube.yml`, or something).